bindings/python/src/ffi.rs

```diff
@@ -0,0 +1,10 @@
+use std::os::raw::c_char;
+
+/// ob_type is not exposed by default as is needed to check if something inherits from a enum
+#[repr(C)]
+#[derive(Debug, Copy, Clone)]
```

Comment:
I assume we don't need `Copy` & `Clone` impls? If not, then I suggest removing them